src/doc_table.h

```diff
@@ -182,11 +192,12 @@ int DocTable_Replace(DocTable *t, const char *from_str, size_t from_len, const c
                      size_t to_len);
 
 /* don't use this function directly. Use DMD_Decref */
-void DMD_Free(RSDocumentMetadata *);
+void DMD_Free(const RSDocumentMetadata *);
```

Comment:
can we remove this one from the `h` file and make it static?

Reply:
I can, but `DMD_Decref` could not be static inline anymore